docs/faq.rst

```diff
@@ -758,6 +763,9 @@ This has some pros and cons, though:
   machine (if you share same backup repo between multiple machines) or if
   your local chunks cache was lost somehow.
 
+Above workaround is necessary only on the run when cache is being recreated,
+and once it completes may be reverted, with borg cache behaving as previously.
+
```

Comment:
Not quite.

If you do that workaround at any time, it afterwards will never use a lot of disk space because it will never create the chunks.archives.d/ cache.

Reply:
I made an experiment, and I blew away cache of all three repositories I have for one of the hosts, and also removed the contents of one of those repositories. I created the removed one anew, for the two remaining repos I started a borg operation that would create the cache (I think it was a `check`), stopped it with `^C`, applied the workaround in the cache directory by removing the `chunks.archive.d` and making it a file as described. I then proceeded to use all the repositories as normal. After a while, in both cache directories, I removed the file `chunks.archive.d` and made it into a directory. With normal usage of borg that directory remained empty.

That's what I'm trying to say by "workaround is necessary when cache is recreated and can be removed afterwards". 

I don't know in what situations `chunks.archive.d/` directory may be used, thus me trying to put it in simple terms that would relate my experience.

Reply:
I've updated my merge request to talk more about this.